### PR TITLE
[ISSUE-#1] chore: initial next.js setup

### LINKED ISSUE #1: Create a front-end code library for development (any front-end framework can be used)

### PR DESCRIPTION
**Pull request by stu01509**
## Description

RESOLVED #1

 Initial next.js setup

## Changes

- Initial next.js setup.
- Used  `pnpm` as package manager.

## Screenshots

image

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+@stu01509 @emberow
```
Comment (stu01509) [inline .github/CODEOWNERS:1]: Should I add Alex as a Code Owner?

**Comment by emberow** [comment]
Screenshot 2024-06-19 at 9 32 21 PM

I have confirmed that this program can work

**Comment by emberow** [comment]
You did a good job.

File: .github/CODEOWNERS (hunk 1 above)
Comment (emberow) [reply .github/CODEOWNERS:1]: Ok, but he is so busy.

**Pull request merged**